src/classes/HiFiAudioAPIData.ts

```diff
@@ -135,12 +138,52 @@ export class HiFiAudioAPIData {
      * ✔ The client sends `hiFiGain` data to the server when `_transmitHiFiAudioAPIDataToServer()` is called.
      * 
      * ✔ The server sends `hiFiGain` data to all clients connected to a server during "peer updates".
+
+     * @param userAttenuation This value affects how far a user's sound will travel in 3D space, without affecting the user's loudness.
+     * By default, there is a global attenuation value (set for a given space) that applies to all users in a space. This default space
+     * attenuation is usually 0.5, which represents a reasonable approximation of a real-world fall-off in sound over distance.
+     * Lower numbers represent less attenuation (i.e. sound travels farther); higher numbers represent more attenuation (i.e. sound drops
+     * off more quickly).
+     * 
+     * When setting this value for an individual user, the following holds:
+     *   - Positive numbers should be between 0 and 1, and they represent a logarithmic attenuation. This range is recommended, as it is
+     * more natural sounding.  Smaller numbers represent less attenuation, so a number such as 0.2 can be used to make a particular 
+     * user's audio travel farther than other users', for instance in "amplified" concert type settings. Similarly, an extremely 
+     * small non-zero number (e.g. 0.00001) can be used to effectively turn off attenuation for a given user within a reasonably 
+     * sized space, resulting in a "broadcast mode" where the user can be heard throughout most of the space regardless of their location
+     * relative to other users. (Note: The actual value "0" is used internally to represent the default; for setting minimal attenuation, 
+     * small non-zero numbers should be used instead. See also "userRolloff" below.)
+     *   - Negative attenuation numbers are used to represent linear attenuation, and are a somewhat artificial, non-real-world concept. However,
+     * this setting can be used as a blunt tool to easily test attenuation, and tune it aggressively in extreme circumstances.
+     *
+     * If you don't supply an `userAttenuation` when constructing instantiations of this class, `userAttenuation` will be `null` and the
+     * default will be used.
+     * 
+     * ✔ The client sends `userAttenuation` data to the server when `_transmitHiFiAudioAPIDataToServer()` is called.
+     * 
+     * ❌ The server never sends `userAttenuation` data.
+     *
+     * @param userRolloff This value represents the progressive high frequency roll-off in meters, a measure of how the higher frequencies 
+     * in a user's sound are dampened as the user gets further away. By default, there is a global roll-off value (set for a given space), currently 12.5 
```

Comment:
I think we just changed the default to 16m?

Reply:
Yeah, I think @howard-stearns is planning to make that change but maybe hasn't yet? I'll update the docs when the mixer gets updated.

Reply:
Roxanne did it during a break. Not in v1.26.2. https://github.com/highfidelity/audionet-hifi/pull/268/files

Reply:
now in v1.26.3 though, correct @howard-stearns ? I'll update the comment.

Reply:
correct